code/modules/overmap/ships/computers/shuttle.dm

```diff
@@ -63,6 +70,24 @@
 			shuttle.set_destination(possible_d[D])
 		return TRUE
 
+	if(action == "jump_to_overmap" && jump_to_overmap_capable)
+		// get our shuttle
+		var/obj/effect/overmap/visitable/ship/landable/landable = connected
+
+		// get horizon (or whatever other main map)
+		var/obj/effect/overmap/visitable/main = GLOB.map_sectors["1"] ? GLOB.map_sectors["1"] : GLOB.map_sectors[GLOB.map_sectors[1]]
+
+		// get the target overmap turf to put our shuttle on
+		var/map_low = OVERMAP_EDGE
+		var/map_high = SSatlas.current_map.overmap_size - OVERMAP_EDGE
+		var/turf/home = CircularRandomTurfAround(main, 2, map_low, map_low, map_high, map_high)
```

Comment:
probably need to delete hazards on the turf you teleport to (and also around?)

Reply:
done